.travis.yml

```diff
@@ -125,44 +173,66 @@ jobs:
       on:
         branch: master
 
-  - stage: verify-deploy
-    name: "Verify deploy"
+  - stage: Deploy
+    name: "Kind deploy"
     env:
     - COMPONENT=secret-provider
     - CLUSTER=kind
     - REGISTRY=ibmcloud
     install:
     - make install-tools
-    script:
-    - make $CLUSTER || travis_terminate 1;
-    - make cluster-prepare || travis_terminate 1;
-    - make -C $COMPONENT deploy
-  - stage: verify-deploy
-    name: "Verify deploy"
+    script: true
```

Comment:
Why a `deploy` and not directly a script? I also don't think that `deploy.script` is a list IIRC

Reply:
as this will be changed to deployment on a OpenShift or shared cluster in future

Reply:
correct. will fix